phys2bids/phys2bids.py

```diff
@@ -261,7 +261,11 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
 
     # Run analysis on trigger channel to get first timepoint and the time offset.
     # #!# Get option of no trigger! (which is wrong practice or Respiract)
-    phys_in.check_trigger_amount(chtrig, thr, num_timepoints_expected, tr)
+    if std_option == 1:
```

Comment:
```suggestion
    if std_option:
```

Boolean check preferred over integer equality!